data/org.freedesktop.portal.Notification.xml

```diff
@@ -116,9 +116,7 @@
             Since version 2, this is deprecated and should not be used.
             Please use the `themed` or `file-descriptor` option to set an icon.
 
-            This is the same format as a serialized `GBytesIcon
-            <https://docs.gtk.org/gio/class.BytesIcon.html>`_ at the moment,
-            but this interoperability may change in the future.
+            This is the icon bytes into a serialized variant bytes array.
```

Comment:
I now know nothing about the particular encoding of it. Isn't there somewhere we can link to for it?

Reply:
Not that I'm aware of, since technically the implementation is also private, it just _happens_ to match the protocol implementation, reason why I am also changing the tests so that we don't rely on GIcon there, since it's very much platform specific.

Reply:
The best we have is https://docs.gtk.org/gio/method.Icon.to_string.html though